config/locales/client.fi.yml

```diff
@@ -0,0 +1,6 @@
+en:
```

Comment:
I see. Well, the good news is that I obviously couldn't do persistent damage.

This pull request can be deleted (if it wasn't already). The whole Github feels like big maze to me, so I leave it for others until I gain some interest. I'll just keep translating what appears in Transifex